src/runtime_src/core/pcie/driver/linux/xocl/subdev/p2p.c

```diff
@@ -1103,15 +1103,12 @@ static int p2p_get_bar_paddr(struct platform_device *pdev, ulong bank_addr,
 		goto  failed;
 	}
 
-	bank_off = p2p_bar_map(p2p, bank_addr, bank_size,
-		XOCL_P2P_CHUNK_SIZE);
+	bank_off = p2p_bar_map(p2p, bank_addr, bank_size, XOCL_P2P_CHUNK_SIZE);
```

Comment:
should not this be p2p_bar_map(p2p, bank_addr, bank_size, 1); ?

Reply:
you are right. fixed.